repoclosure/yum_el7.conf

```diff
@@ -48,9 +48,9 @@ baseurl=http://yum.puppetlabs.com/el/7/PC1/$basearch/
 name=el7-puppet-5
 baseurl=https://yum.puppetlabs.com/puppet5/el/7/$basearch/
 
-[el7-tfm-ror51]
-name=el7-tfm-ror51
-baseurl=https://copr-be.cloud.fedoraproject.org/results/@theforeman/tfm-ror51/epel-7-$basearch/
+[el7-tfm-ror51-5.1.4]
```

Comment:
Would it make sense to keep this on `el7-tfm-ror51` to keep the repoclosure changes minimal?